common/commands/src/coconut/issue_credentials.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022-2023 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use clap::Parser;
+use nym_issue_credential::utils;
+
+use crate::context::SigningClientWithNyxd;
+use nym_network_defaults::NymNetworkDetails;
+use nym_validator_client::nyxd::Coin;
+
+#[derive(Debug, Parser)]
+pub struct Args {
+    /// Home directory of the client that is supposed to use the credential.
+    #[clap(long)]
+    pub(crate) client_home_directory: std::path::PathBuf,
+
+    /// A mnemonic for the account that buys the credential
+    #[clap(long)]
+    pub(crate) mnemonic: Option<bip39::Mnemonic>,
+
+    /// The amount of utokens the credential will hold. If recovery mode is enabled, this value
+    /// is not needed
```

Comment:
if the value is not needed, set it to an `Option<u64>` so that clap would not try to enforce it (and you'd have a nicer type to work with)

Reply:
Actually this comment is incorrect, it is left over from the credential binary that has the recovery mode. Instead of the recovery mode I am first trying to recover the credentials and then create new ones if nothing is recovered